go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -170,6 +174,13 @@ func (ncm *nodeNetworkControllerManager) Start(ctx context.Context) (err error)
 		}, time.Minute, ncm.stopChan)
 	}
 
+	// Let's create Route manager that will manage routes.
+	ncm.wg.Add(1)
+	go func() {
+		defer ncm.wg.Done()
```

Comment:
err that's not making sense to me @pperiyasamy if we are going to use the default network's route manager why are calling Run here? can you squash this commit into first one? then I can see where in DNCC you are deleting the invocation from